frontend/rolecall/src/app/piece/piece_editor.component.ts

```diff
@@ -43,6 +50,14 @@ export class PieceEditor implements OnInit {
 
   lastSelectedPieceName: string;
 
+  currentType = 0;
+
+  segmentTypes = ["SEGMENT", "PIECE", "REVELATION"];
+  selectedSegmentType: "SEGMENT" | "PIECE" | "REVELATION";
```

Comment:
Would be great to use PieceType to avoid repeating these strings

Reply:
Done.